azerothcore/azerothcore-wotlk
# fix(DB/Spell): Make Amplify Magic Ranks 4-7 as Positive

## Changes Proposed:
-  Correctly set Amplify Magic Ranks 4-7 as Positive Buffs

## Issues Addressed:
- Closes #4565

## Tests Performed:
- Ran SQL, tested on mage.  It fixes the issue.


## How to Test the Changes:
- Create a mage and learn all ranks of Amplify Magic (need lvl 77)
- Cast Amplify Magic rank 4, 5, 6 and 7 on yourself and ensure that they are positive buffs.


## Target Branch(es):
